pyangbind/helpers/identity.py

```diff
@@ -118,11 +120,12 @@ def build_store_from_definitions(self, ctx, defnd):
           # and if not, then push this identity back onto the stack
           unresolved_identities.append(this_id)
           unresolved_identity_count[this_id] += 1
-          if unresolved_identity_count[this_id] > 1000:
-            if unresolved_idc[ident] > 1000:
-              sys.stderr.write("could not find a match for %s base: %s\n" %
-                (iddef.arg, base_name))
-              error_ids.append(ident)            
+          # FIXME(Joey Wilhelm): `unresolved_idc` and `ident` are both undefined. What is the intent of this code?
```

Comment:
I made a comment in the other PR on this -- linking here for completeness: https://github.com/robshakir/pyangbind/pull/160#discussion_r171901037

Reply:
Would it be alright to leave this fix for the py3 branch, so that this branch can stay as just linting?

Reply:
Sure - that works for me.